src/lib/seam/components/SupportedDeviceTable/SupportedDeviceContent.tsx

```diff
@@ -14,19 +14,22 @@ interface SupportedDeviceContentProps {
   resetFilterValue: () => void
   filters: DeviceModelFilters
   brands: string[] | null
+  omitBrands: string[]
```

Comment:
What about `disabledBrands`, or `hiddenBrands`?

Reply:
I like `hideBrands`

Reply:
that still sounds verb-y though. 🤷‍♂️

Reply:
I considered both. It can't be disable since that would be a boolean props. It could be `disabledBrands`. Actually, omit is a verb, it should be omitted brands, but that's awkward.

Maybe `hiddenBrands`?

Reply:
`disabledBrands` is nice. We use the word 'disable' elsewhere, so it's nice to use familiar vocabulary

Reply:
Yes, but in this case `disable` and `disabled` is easy to mess up / misread (one letter), and disable is always used for bool props.

Reply:
@dawnho What about `excludedBrands`. I think include / exclude is a standard term.